manifest.yml

```diff
@@ -14,5 +14,6 @@ applications:
     GOVUK_APP_DOMAIN: cloudapps.digital
     GOVUK_WEBSITE_ROOT: www.gov.uk
     BASIC_AUTH_USERNAME: gds
+    REDIS_PROVIDER: http://govuk-coronavirus-business-volunteer-form-redis.apps.internal:8080
```

Comment:
I don't think this URL will work. The redis URL will be provided to the app as part of the VCAP_SERVICES environment variable. `apps.internal` only works for other apps.

Reply:
Ah thanks, I wasn't sure about this. I've amended the commit to use VCAP_SERVICES.